hadoop-ozone/recon/pom.xml

```diff
@@ -331,6 +331,11 @@
       <artifactId>bonecp</artifactId>
       <version>0.8.0.RELEASE</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.derby</groupId>
+      <artifactId>derby</artifactId>
+      <version>10.14.2.0</version>
+    </dependency>
     <dependency>
```

Comment:
Thanks the patch @avijayanhwx 

Can we remove old sqlite dependency from here and from `hadoop-ozone/dist/src/main/license/LICENSE.txt`?

Is it still required?

One other question: do you have any performance numbers between sqlite vs derby? (I don't think it's a big problem, just interested if you know sg.)

Reply:
@elek Thanks for the review.  We have not done perf testing on our SQL DB yet since we don't have a scale issue there (hopefully :) ) We will take up a task to try perf analysis between the 2 DBs. Until then, we can assume the performance between Derby vs Sqlite is similar as seen in some online references. 

Regarding the sqlite dependency that has been left around, I wanted to retain the out of the box support for sqlite in Recon (without JAR being supplied by the user). Also, we have sqlite JAR dependency in the unit tests.